tests/unit/generateNotificationData.unitTest.ts

```diff
@@ -18,8 +18,8 @@ describe("notificationData", () => {
     context("when parsing the visit and the test results", () => {
       it("should return a correct test stations emails", () => {
         const testResultsArray = TestResultsService.prototype.expandTestResults(JSON.parse(testResultsList.body));
-        testResultsArray[0].testStartTimestamp = "2019-01-14T10:05:16.987Z";
-        testResultsArray[0].testEndTimestamp = "2019-01-14T10:40:02.987Z";
+        testResultsArray[0].testTypes.testTypeStartTimestamp = "2019-01-14T10:05:16.987Z";
+        testResultsArray[0].testTypes.testTypeEndTimestamp = "2019-01-14T10:40:02.987Z";
```

Comment:
I think this will work the same way anyway, whether you sort by testStartTimestamp or testTypeStartTimestamp - the order will be identical. The testStartTimestamp for the TR2 TT1 will always be after TR1 TT1, in the same way TR2 will always be after TR1.

- 09:00 Test Result 1 (testStartTimestamp)
---- TR 1 TestType1 09:01 - 09:35 (testTypeStartTimestamp)
---- TR 1 TestType2 09:36 - 09:55 testTypeStartTimestamp)

-10:00 Test Result 2 (testStartTimestamp)
---- TR 2 TestType1 10:05-10:20 (testTypeStartTimestamp)
---- TR 2 TestType2 10:25 - 11:00 (testTypeStartTimestamp)